.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  npgsql-dev:
+    image: mcr.microsoft.com/dotnet/sdk:5.0
+    volumes:
+      - ..:/workspace:cached
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+    links: 
+      - db
+
+  db:
+    image: postgres
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes:
+      - ./init-db.sh:/docker-entrypoint-initdb.d/init-db.sh
+    ports: 
+      - 5432:5432
+
+  pgadmin:
+    image: dpage/pgadmin4
+    environment:
+      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
+      PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
+    ports:
+      - 5050:80
+    links:
+      - db
```

Comment:
Out of curiosity, it this required? I mean, what if I do not use pgAdmin at all, but have another tool to connect to the database?

Reply:
I personally don't mind that it's there by default - I'm assuming the overhead is negligible for the environment...

Reply:
Yes, it is optional but I still do not manage to connect to it from dev host 😞
I have requested for some help from some devcontainer experts...